src/main/java/com/example/flanki/FlankiApplication.java

```diff
@@ -0,0 +1,11 @@
+package com.example.flanki;
```

Comment:
What about using other package than default one?
For example pl.lodz.p.edu.flanki?